defaults/imagedigests.json

```diff
@@ -0,0 +1,43 @@
+{
+  "quay.io/cilium/cilium": {
+    "v1.8.13": "@sha256:070a57faa72ca55b045861453a2f1697e4d582a75cf2b24937e0397684abcb3f",
+    "v1.9.11": "@sha256:47f923325069a697d5baf5314c7fe936bdf34e7c8154666e6762a78be1ddc3ec",
+    "v1.10.5": "@sha256:0612218e28288db360c63677c09fafa2d17edda4f13867bcabf87056046b33bb",
```

Comment:
Should we switch this to 1.10.6 which was released last week? Same for the `operator-*`, `clustermesh-apiserver` and `hubble-relay` images.

Reply:
Added.